pkg/services/tree/service.go

```diff
@@ -319,6 +319,34 @@ func (s *Service) Apply(_ context.Context, req *ApplyRequest) (*ApplyResponse, e
 	})
 }
 
+func (s *Service) GetOpLog(req *GetOpLogRequest, srv TreeService_GetOpLogServer) error {
+	b := req.GetBody()
+	cid := getCID(b.GetContainerId())
+
+	h := b.GetHeight()
+	for {
+		lm, err := s.forest.TreeGetOpLog(cid, b.GetTreeId(), h)
+		if err != nil || lm.Time == 0 {
```

Comment:
Here I guess we expect `forest.TreeGetOpLog` to return `pilorama.Move{}, nil` if log op is not found. I only understood it from the implementation, so I suggest to clarify this return in interface method docs.

Reply:
Fixed.